debian/control

```diff
@@ -0,0 +1,30 @@
+Source: count-distinct
+Section: database
+Priority: optional
+Maintainer: Ilya Peretyagin <ilya@brandwatch.com>
+Build-Depends: debhelper (>= 9), postgresql-server-dev-all (>= 180)
+Standards-Version: 3.9.8
+Homepage: https://github.com/tvondra/count_distinct
+Vcs-Git: git@github.com:bashtanov/count_distinct.git
+Vcs-Browser: https://github.com/bashtanov/count_distinct
+
+Package: postgresql-count-distinct-9.4
+Architecture: any
+Depends: postgresql-9.4, ${shlibs:Depends}, ${misc:Depends}
+Description: COUNT_DISTINCT aggregate
+ This extension provides an alternative to COUNT(DISTINCT ...) which for
+ large amounts of data often ends in sorting and poor performance.
+ .
+ This package contains the pg_repack program and the server extension for
+ PostgreSQL 9.4.
+
+Package: postgresql-count-distinct-9.6
```

Comment:
for 9.6 it would be better to use https://github.com/tvondra/count_distinct/tree/master, probably with `default_version` bumped to 3.0.0 in `count_distinct.control`